ophyd/signal.py

```diff
@@ -18,7 +18,7 @@
 
 
 class Signal(OphydObject):
-    '''A signal, which can have a read-write or read-only value.
+    r'''A signal, which can have a read-write or read-only value.
```

Comment:
Is there a reason to make docstrings raw strings? Or is this just for the contents of this particular docstring?

Reply:
Looks like there are some invalid escape sequences in this specific docstring, according to @tacaswell (cherry picked his commit)

Reply:
:+1: